overops.properties

```diff
@@ -0,0 +1,14 @@
+category=Integration
+description=Identify and drill down into runtime errors in Java applications
+homepageUrl=https://github.com/takipi-field/sonar-plugin-overops
+archivedVersions=
+publicVersions=1.0.3
+
+defaults.mavenGroupId=com.overops.plugins.sonar
+defaults.mavenArtifactId=overops-plugin
+
+1.0.3.description=Initial Release
+1.0.3.sqVersions=[7.9,LATEST]
```

Comment:
Your docs say compatibility starts at 7.9.1...?

Reply:
Yes, thanks! I've confirmed the correct version is 7.9.1 and updated it here.